### PR TITLE
Merge into AtomLinter

### LINKED ISSUE #50: Object.activate is deprecated.
AtomLinter v0.X.Y API has been deprecated. Please refer to the Linter docs to update and the latest API: https://github.com/AtomLinter/Linter/wiki/Migrating-to-the-new-API
```
Object.activate (/Users/egon/.atom/packages/linter/lib/main.coffee:57:8)
Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:240:19)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:221:30)
Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:165:15)
Package.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:213:14)
PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:452:21)
```

### LINKED ISSUE #51: Doesn't work on Atom 1.0.0
I have installed linter-rubocop and gave the rubocop path and it is not working on Atom 1.0.0. I am using linter-ruby alternatively.

### LINKED ISSUE #52: LinterRubocop.Linter is deprecated.
AtomLinter v0.X.Y API has been deprecated. Please refer to the Linter docs to update and the latest API: https://github.com/AtomLinter/Linter/wiki/Migrating-to-the-new-API
```
LinterRubocop.Linter (/Users/josh/.atom/packages/linter/lib/linter.coffee:60:4)
new LinterRubocop (/Users/josh/.atom/packages/linter-rubocop/lib/linter-rubocop.coffee:25:4)
Object.lint (/Users/josh/.atom/packages/linter/lib/legacy.coffee:61:21)
<unknown> (/Users/josh/.atom/packages/linter/lib/editor-linter.coffee:69:25)
currentLinter (/Users/josh/.atom/packages/linter/lib/editor-linter.coffee:68:19)
<unknown> (/Users/josh/.atom/packages/linter/lib/editor-linter.coffee:82:22)
```

### PR DESCRIPTION
**Pull request by k2b6s9j**
With the permission of @caseywebdev I would like to merge this fork into the main repo, and allow @caseywebdev to maintain this repo with the permission of @AtomLinter/owners.

Fixes #50 
Fixes #51 
Fixes #52

### CODE REVIEW

**Comment by k2b6s9j** [comment]
Before this can happen @caseywebdev must give his approval, and one of @AtomLinter/owners must give their approval to bring @caseywebdev as part of the team.

**Comment by steelbrain** [comment]
It's always nice to have new people in the team! :)

**Comment by caseywebdev** [comment]
That's cool with me :+1:

**Comment by DarthSim** [comment]
There is a bug. I use rbenv, and I have rubocop-rspec installed only for one (not global) ruby version. When I'm trying to lint a file, I'm getting an error because rubocop can't load rubocop-rspec. To fix this issue, you need to add `cwd` option to BufferedProcess call:

```javascript
{join, resolve, dirname} = require 'path'

...

new BufferedProcess
  command: ...
  args: ...
  options:
    cwd: dirname(editor.getPath())
```

File: package.json [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
-  "name": "linter-rubocop",
-  "linter-package": true,
+  "name": "linter-rubocop-caseywebdev",
   "activationCommands": [],
-  "main": "./lib/init",
-  "version": "0.2.7",
+  "version": "0.2.9",
   "description": "Lint `Ruby` on the fly, using rubocop",
-  "repository": "https://github.com/AtomLinter/linter-rubocop",
+  "repository": "https://github.com/caseywebdev/linter-rubocop",
```
Comment (AsaAyers) [inline package.json]: This should probably be changed back

**Comment by AsaAyers** [comment]
@caseywebdev Welcome! I invited you to a new team for managing this repo.

I'll leave the merging to someone who knows what rubocop is :stuck_out_tongue_winking_eye:

**Comment by k2b6s9j** [comment]
@DarthSim Could you file that under an issue here?

I'll be merging this in and fixing the url now.

**Comment by k2b6s9j** [comment]
@caseywebdev I'll leave it to you to delete your forked package, and publish this updated one.

**Comment by DarthSim** [comment]
@k2b6s9j I hoped you could fix that before merge. I'll create PR.

**Comment by k2b6s9j** [comment]
@DarthSim this isn't actually my code. This is @caseywebdev's code, this PR just merged his forked repo in and proposed making him maintainer.

**Pull request merged**